applets/006-tcpdump/applet.py

```diff
@@ -139,8 +140,9 @@ def tcpdump_stop(x, arg_d):
 
 def main(x, conf, args):
     if not len(args) >= 4:
-        x.p.msg("wrong usage. use: [hostname] id:[number] mode:[start|stop] "
-                "local-file-name:\"[filename]\" filter:\"[filter]\"\n")
+        x.p.err("wrong usage. use: [hostname] id:[number] mode:[start|stop]\n"
+                "mode:start -> \tfilter:\"[filter]\"\n"
```

Comment:
Good question. In fact, filter CAN be empty, and EVERYTHING on the interface will be capped.